src/bidiMapper/domains/network/NetworkRequest.ts

```diff
@@ -210,6 +211,29 @@ export class NetworkRequest {
       requestId: fetchRequestId,
       errorReason,
     });
+
+    // TODO: mark is as non-blocked?
+    // this.#isBlocked = false;
```

Comment:
Is the request still blocked.

Reply:
It shouldn't be, as at that point we have "continued" it. I just uncommented that line to reflect so.